taipy/gui/viselements.json

```diff
@@ -1128,6 +1128,12 @@
                         "type": "str",
                         "doc": "The type of the gauge.<br/>Possible values are:\n<ul>\n<li>\"none\"</li>\n<li>\"circular\"</li>\n<li>\"linear\"</li></ul>."
                     },
+                    {
+                        "name": "title",
+                        "default_value": "None",
```

Comment:
@FabienLelaquais Why do we have to specify a default value when there is none

Reply:
None is different from None.
Hmmm let me rephrase that.

If you don't specify a default value, then the property is required.
If you do specify a default value, even if it is "None", then the property is optional, with this value as the default.